frontend/shared/src/client/sites.ts

```diff
@@ -0,0 +1,219 @@
+import {client} from './client'
+import {
+  AddSiteRequest,
+  ListSitesRequest,
+  ListSitesResponse,
+  DeleteSiteRequest,
+  SitesClientImpl,
+} from './.generated/daemon/v1alpha/sites'
+import {
+  SiteClientImpl,
+  CreateInviteTokenRequest,
+  BlockAccountRequest,
+  DeleteMemberRequest,
+  GetMemberRequest,
+  Member_Role,
+  ListMembersRequest,
+  GetSiteInfoRequest,
+  ListBlockedAccountsRequest,
+  RedeemInviteTokenRequest,
+  UnblockAccountRequest,
+  UpdateSiteInfoRequest,
+  PublishRequest,
+  UnpublishRequest,
+} from './.generated/site/v1alpha/site'
+import {ListPublicationsRequest} from './.generated/documents/v1alpha/documents'
+
+export async function addSite(hostname: string, inviteToken?: string) {
+  return await new SitesClientImpl(client).addSite(
+    AddSiteRequest.fromPartial({
+      hostname,
+      inviteToken,
+    }),
+  )
+}
+
+export async function listSites(): Promise<ListSitesResponse> {
+  // EV TESTING:
+  // return {
+  //   sites: [
+  //     {
+  //       hostname: 'temp-test.org',
+  //       role: Member_Role.OWNER,
+  //     },
+  //   ],
+  //   nextPageToken: '',
+  // }
+  return await new SitesClientImpl(client).listSites(
+    ListSitesRequest.fromPartial({}),
+  )
+}
+
+export async function deleteSite(hostname: string) {
```

Comment:
IMO this name is confusing because the site is being removed from the local app, not deleted

but I don't care if you feel strongly about this

Reply:
Hmm. I see what you mean! Although I could also think about it as "deleting the site configuration from the local app".

The thing is that I've been following these [API Guidelines](https://google.aip.dev) as much as I could, and I've been doing that for years when working with GRPC, so I'm a bit biased :) They define a set of standard API methods, and one of them is [delete](https://google.aip.dev/135). So for coherence with the rest of the APIs we have, I figured it's better to use Delete here. In the end I believe both words are interchangeable and open to interpretation in many ways.